qrust-api/src/main/resources/application-harvey.yml

```
@@ -0,0 +1,21 @@
+      ddl-auto: create
+    properties:
+      hibernate:
+        #        show_sql: true
+        format_sql: true
+        highlight_sql: true
+        default_batch_fetch_size: 1000 #최적화 옵션
+    open-in-view: false #OSIV 를 true로 설정하면, request가 반환될 때 까지 커넥션을 보유
+
+logging.level:
+  org.hibernate.SQL: debug # 이거 끄면 format_sql 적용 안됨
+#  org.hibernate.type: trace
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)